README.md

```diff
@@ -121,30 +121,36 @@ This example defines 3 routes:
 // src/App.js
 
 import React, { Component } from 'react';
-import { BrowserRouter as Router, Route } from 'react-router-dom';
-import { SecureRoute, Security, LoginCallback } from '@okta/okta-react';
+import { BrowserRouter as Router, Route, withRouter } from 'react-router-dom';
+import { Security, LoginCallback, useOktaAuth } from '@okta/okta-react';
+import _SecureRoute from '@okta/okta-react/src/SecureRoute';
 import { OktaAuth } from '@okta/okta-auth-js';
 import Home from './Home';
 import Protected from './Protected';
 
+const SecureRoute = (props) => <_SecureRoute useOktaAuth={useOktaAuth} {...props} />;
```

Comment:
I'm not following why _SecureRoute exists just so that you can create SecureRoute.  What am I missing?

Reply:
Idea was to create decorator to let `SecureRoute` to use okta context. I've removed excess component creation.